<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(154, 153, 150);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0); color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>4</y>
      <width>921</width>
      <height>681</height>
     </rect>
    </property>
    <widget class="QWidget" name="homePage">
     <widget class="ClickableLabel" name="batteryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 209, 122);</string>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>12:40 - 14 Nov</string>
      </property>
     </widget>
     <widget class="QLabel" name="reservoirLabel">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>20</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>235 u</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="glucoseGraph" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>600</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 56, 70);</string>
      </property>
     </widget>
     <widget class="QLabel" name="glucoseLabel">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>100</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>10.0 mmol/L</string>
      </property>
     </widget>
     <widget class="QLabel" name="durationLabel">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>140</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(119, 118, 123);</string>
      </property>
      <property name="text">
       <string>3 HRS</string>
      </property>
     </widget>
     <widget class="QLabel" name="iobLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>INSULIN ON BOARD: 2.2 u</string>
      </property>
     </widget>
     <widget class="QPushButton" name="optionsButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: #3a3a3a; color: orange; font-weight: bold; border-radius: 6px; padding: 8px; } QPushButton:pressed { background-color: #2b2b2b; }</string>
      </property>
      <property name="text">
       <string>OPTIONS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bolusButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: #3a3a3a; color: #1e90ff; font-weight: bold; border-radius: 6px; padding: 8px; } QPushButton:pressed { background-color: #2b2b2b; }</string>
      </property>
      <property name="text">
       <string>BOLUS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonViewLog">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(94, 92, 100)</string>
      </property>
      <property name="text">
       <string>View Log</string>
      </property>
     </widget>
     <widget class="QLabel" name="basalRateLabel">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>200</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 163, 72);</string>
      </property>
      <property name="text">
       <string>Basal: __ u/hr</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bolusPage">
     <widget class="QLabel" name="labelBolusTitle">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(153, 193, 241)</string>
      </property>
      <property name="text">
       <string> Bolus Delivery</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditGlucose">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCarbs">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="targetGlucoseInput_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="isfInput_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="crInput_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDeliver">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 132, 228);</string>
      </property>
      <property name="text">
       <string>Deliver</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackFromBolus">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 193, 241);</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Glucose:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Carbs:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Target Glucose:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ISF:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>270</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Carb Ratio:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSaveProfile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 97, 203);</string>
      </property>
      <property name="text">
       <string>Profiles</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(119, 118, 123)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxProfiles">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 191, 188);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="optionsPage">
     <widget class="QLabel" name="labelOptionsTitle">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(98, 160, 234);</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBasalRate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>142</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonUpdateBasal">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 132, 228)</string>
      </property>
      <property name="text">
       <string>Update Basal Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBasalStatus">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackFromOptions">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(98, 160, 234);</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Basal Rate:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxControlIQ">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(98, 160, 234);</string>
      </property>
      <property name="text">
       <string>Enable Control IQ</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
